ui/app/templates/components/alert-popup.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="message {{type.class}}">
+  <div class="columns is-mobile is-variable is-1">
+    <div class="column is-narrow message-icon">
+      <ICon @glyph="{{type.glyph}}" @size="20" @excludeIconClass={{true}} />
+    </div>
+    <div class="column">
+      <button type="button" class="close-button" {{action close}}>
+        {{i-con excludeIconClass=true glyph="close" aria-label="Close"}}
```

Comment:
Can we move this to angle bracket while we're here? Not sure why it wasn't already in the cluster template considering the angle bracket one above :-/.